### PR TITLE
Fix mask-border-* props for new ff

### PR DESCRIPTION
**Pull request by AleshaOleg**

### CODE REVIEW

File: karma.conf.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ const travisJobNumber = process.env.TRAVIS_JOB_NUMBER
 module.exports = (config) => {
   config.set({
     customLaunchers: browsers,
-    browsers: ['Chrome'],
+    browsers: ['Firefox'],
```
Comment (kof) [inline karma.conf.js]: ?

Comment (AleshaOleg) [reply karma.conf.js]: forgot to revert after local tests

**Comment by AleshaOleg** [comment]
Looks like, here the same situation as with `writing-mode` prop. In older browsers, we should prefix this prop, and in newest (> FF 52), we shouldn't. https://caniuse.com/#search=mask-border

Need to find complex solutions for such props as this two.
For now, I fix tests, to make them pass in local FF tests.

**Comment by kof** [comment]
What does autoprefixer?

**Comment by AleshaOleg** [comment]
https://github.com/postcss/autoprefixer/issues/502

**Comment by kof** [comment]
We can either do the same or just skip it.

**Pull request merged**